pkg/util/errutil/errors.go

```diff
@@ -41,6 +41,23 @@ func NewBase(reason StatusReason, msgID string, opts ...BaseOpt) Base {
 	return b
 }
 
+func ErrorFrom(e error) (*Error, error) {
+	fmt.Printf("ErrorFrom: %e\n", e)
+	//nolint:errorlint
+	o, isGrafanaError := e.(Error)
+	if isGrafanaError {
+		return &o, nil
+	}
+	//nolint:errorlint
+	base, isBase := e.(Base)
```

Comment:
The linter is correct for this function, you'll want to use `errors.As` here, otherwise it won't unwrap errors 🙂